EnglishFilter/sections/foreign.txt

```diff
@@ -791,6 +791,11 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33593
 kkphm.com,haohaiyo.cc,51kkp.club##a[href*="kkpmh.com/"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/33851
+flyblog.cc##iframe[src*="ifoodie.tw/"]
```

Comment:
Unsure about this one, looks like annoyance to me.

Reply:
or not even annoyance

Reply:
Well, I deleted this rule.